operator/roles/forkliftcontroller/tasks/main.yml

```diff
@@ -79,6 +79,23 @@
       definition: "{{ lookup('template', 'controller/deployment-controller.yml.j2') }}"
       merge_type: "merge"
 
+  - name: "Remove transfer network annotation when disabled"
+    k8s:
+      definition:
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: "{{ controller_deployment_name }}"
+          namespace: "{{ app_namespace }}"
+        spec:
+          template:
+            metadata:
+              annotations:
+                k8s.v1.cni.cncf.io/networks: null
+      state: present
+      merge_type: "strategic-merge"
+    when: controller_transfer_network | default('') == ''
+
```

Comment:
Please checkout other jinja templates how they deal with the empty variables

Reply:
https://github.com/kubev2v/forklift/blob/9e98d71a145fd183865f01a7330b614b0eb085ec/operator/roles/forkliftcontroller/templates/controller/deployment-controller.yml.j2#L50-L53